pkg/util/fixtures/fixtures.go

```diff
@@ -10,12 +10,12 @@ func NewTestOpenShiftCluster() *api.OpenShiftManagedCluster {
 		Name:     "openshift",
 		Config:   &api.Config{},
 		Properties: &api.Properties{
-			FQDN:             "www.example.com",
+			FQDN:             "www.example.eastus.cloudapp.azure.com",
```

Comment:
will necessarily be foo.region.cloudapp.azure.com - can validate the number of domain parts